viewer/core/src/public/RevealManager.ts

```diff
@@ -201,13 +201,6 @@ export class RevealManager {
     modelIdentifier: ModelIdentifier,
     options?: AddCadModelOptions
   ): Promise<PointCloudNode | CadNode> {
-    MetricsLogger.trackLoadModel(
```

Comment:
Why it was removed from here entirely but not updated to a new format?

Reply:
This was moved to `CadModelFactory` and `PointCloudManager`, because `RevealManager` doesn't know about formatVersion as this needs to be fetched from the server first. A bit annoying to have this two places, but acceptable.